### PR TITLE
Remove node 6, use prebuildify fork for now, build on node 8

### PR DESCRIPTION
**Pull request by ralphtheninja**
See https://github.com/Level/leveldown/issues/553#issuecomment-447646918

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "node-gyp": "^3.8.0",
     "nyc": "^12.0.2",
     "optimist": "~0.6.1",
-    "prebuildify": "^2.9.1",
+    "prebuildify": "ralphtheninja/prebuildify",
```
Comment (ralphtheninja) [inline package.json]: Use this fork for now until PR has been merged/released.

File: package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "test": "standard && nyc tape test/*-test.js",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "rebuild": "node-gyp rebuild",
-    "prebuild": "prebuildify --napi --strip",
+    "prebuild": "prebuildify -t $(node -v) -t electron@3.0.0 --napi --strip",
```
Comment (ralphtheninja) [inline package.json]: Not sure if `$(node -v)` is kosher for windows.

Comment (vweevers) [reply package.json]: It is not, unless we switch to travis for windows ;)

Comment (vweevers) [reply package.json]: Perhaps move this logic to `travis.yml` and `appveyor.yml`

Comment (ralphtheninja) [reply package.json]: Hmm, but we do `npm run prebuild` from those files. We could hardcode the node version number for now and pull this back in after we switch to travis for windows?

Comment (vweevers) [reply package.json]: We could do `npm run prebuild -- -t ..`. A bit ugly. I'm fine with hardcoding 👍

Comment (ralphtheninja) [reply package.json]: Another thing, it seems that appveyor tries to do `npm run prebuild` https://ci.appveyor.com/project/peakji/leveldown/builds/21045244/job/rvmujh6v9kqrjdfd#L2339 , which it shouldn't?

File: appveyor.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ test_script:
   - npm test
 
 after_test:
-  - ps: If ($env:nodejs_version -eq "Current") { npm run prebuild }
+  - ps: If ($env:nodejs_version -eq "8") { npm run prebuild }
```
Comment (ralphtheninja) [inline appveyor.yml]: This seems to be run unconditionally. Not sure if that's ok or if we should add e.g. `version-unchanged` to `package.json`.

Comment (vweevers) [reply appveyor.yml]: Might be good to keep this, to know that prebuilds are working (and not find out they're broken on release)? idk

Comment (ralphtheninja) [reply appveyor.yml]: Aye, lets just get this ball rolling first and tweak as we go. Still kind of figuring this out.

Comment (vweevers) [reply appveyor.yml]: If we don't want this behavior, then an `on: ..` could work (same as under `deploy:`)

**Comment by ralphtheninja** [comment]
I'll let appveyor finish up before merging.

**Comment by ralphtheninja** [comment]
And also needs rebasing ;)

**Comment by ralphtheninja** [comment]
Looks like we got some error on appveyor even though the build passed

screenshot from 2018-12-16 18-31-46

**Comment by ralphtheninja** [comment]
Just checked old sodium-native setup on appveyor and it shows similar output:

screenshot from 2018-12-16 18-37-28

From https://ci.appveyor.com/project/mafintosh/sodium-native/builds/20896258/job/ql6fno6kfrwi3rs2

**Comment by ralphtheninja** [comment]
Merged too early, current got canceled on appveyor.

**Pull request merged**